src/pages/DataSubmissionForm.js

```diff
@@ -159,13 +159,15 @@ export const DataSubmissionForm = (props) => {
           <img id='lock-icon' src={lockIcon} style={Styles.HEADER_IMG} />
         </div>
         <div style={Styles.HEADER_CONTAINER}>
-          <h1>Register a Dataset</h1>
-          <div style={{fontSize: '1.6rem'}}>
-            Submit a new dataset to DUOS
+          <div style={Styles.TITLE}>
+            Study Registration Form
+          <div style={Styles.MEDIUM_DESCRIPTION}>
+            Submit new datasets to DUOS
           </div>
         </div>
       </div>
     </div>
+    </div>
```

Comment:
The codacy failure is partly due to the indentation in this block. The entire div should look more like this:
```
    <div style={{ display: 'flex', justifyContent: 'space-between', width: '112%', marginLeft: '-6%', padding: '0 2.5%' }}>
      <div className='left-header-section' style={Styles.LEFT_HEADER_SECTION} >
        <div style={Styles.ICON_CONTAINER}>
          <img id='lock-icon' src={lockIcon} style={Styles.HEADER_IMG}  alt={'lock-icon'}/>
        </div>
        <div style={Styles.HEADER_CONTAINER}>
          <div style={Styles.TITLE}>
            Study Registration Form
            <div style={Styles.MEDIUM_DESCRIPTION}>
            Submit new datasets to DUOS
            </div>
          </div>
        </div>
      </div>
    </div>
```